components/public-api-server/pkg/proxy/errors.go

```diff
@@ -43,15 +43,28 @@ func categorizeRPCError(err error) *connect.Error {
 		// components/gitpod-protocol/src/messaging/error.ts
 		case 409:
 			return connect.NewError(connect.CodeAlreadyExists, fmt.Errorf(rpcErr.Message))
+		// components/gitpod-protocol/src/messaging/error.ts - SETUP_REQUIRED (no user)
```

Comment:
`FailedPrecondition`?

Reply:
I will revert these 2 since they are not related to VS Code Desktop used APIs.